src/Components/Lobby/Lobby.js

```diff
@@ -1,25 +1,26 @@
 import { Component } from 'react';
 import { Link } from 'react-router-dom';
 import './Lobby.scss';
-
+import PropTypes from 'prop-types'
 
 class Lobby extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      mood: '',
+
     }
   }
+
   render() {
     return (
       <section className='Lobby'>
-        <h1>Welcome Back, it's TODAY!</h1>
+        <h1>Welcome Back, it's {new Date().toDateString()}</h1>
```

Comment:
I'm glad we figured out a way to get a legible date into the site without having to implement moment. I know we could without too much trouble, but I like not having to mess with installing packages unless we absolutely have to.